dist/amd/index-internal.d.ts

```diff
@@ -1,11 +1,11 @@
-import IredirectHandler from "./iredirect-handler";
-import OpenIdConnectAuthorizeStep from "./open-id-connect-authorize-step";
-import OpenIdConnectConfigurationManager from "./open-id-connect-configuration-manager";
-import OpenIdConnectFactory from "./open-id-connect-factory";
-import OpenIdConnectLogger from "./open-id-connect-logger";
-import OpenIdConnectNavigationStrategies from "./open-id-connect-navigation-strategies";
-import OpenIdConnectRouting from "./open-id-connect-routing";
-import OpenIdConnectUserBlock from "./open-id-connect-user-block";
-import OpenIdConnectUserDebug from "./open-id-connect-user-debug";
-export * from "./internal-types";
-export { IredirectHandler, OpenIdConnectAuthorizeStep, OpenIdConnectConfigurationManager, OpenIdConnectFactory, OpenIdConnectLogger, OpenIdConnectNavigationStrategies, OpenIdConnectRouting, OpenIdConnectUserBlock, OpenIdConnectUserDebug };
+import { RedirectHandler } from './redirect-handler';
+import { OpenIdConnectAuthorizeStep } from './open-id-connect-authorize-step';
+import { OpenIdConnectConfigurationManager } from './open-id-connect-configuration-manager';
+import { OpenIdConnectFactory } from './open-id-connect-factory';
+import { OpenIdConnectLogger } from './open-id-connect-logger';
+import { OpenIdConnectNavigationStrategies } from './open-id-connect-navigation-strategies';
+import { OpenIdConnectRouting } from './open-id-connect-routing';
+import { OpenIdConnectUserBlock } from './open-id-connect-user-block';
+import { OpenIdConnectUserDebug } from './open-id-connect-user-debug';
+export * from './internal-types';
+export { RedirectHandler, OpenIdConnectAuthorizeStep, OpenIdConnectConfigurationManager, OpenIdConnectFactory, OpenIdConnectLogger, OpenIdConnectNavigationStrategies, OpenIdConnectRouting, OpenIdConnectUserBlock, OpenIdConnectUserDebug };
```

Comment:
Could you please remove the `dist` directory from the pull request. That will make it easier to review.

Reply:
will do, tomorrow